gtsam_unstable/slam/SmartProjectionPoseFactorRollingShutter.h

```diff
@@ -86,19 +84,40 @@ class SmartProjectionPoseFactorRollingShutter
   /**
    * Constructor
    * @param Isotropic measurement noise
+   * @param cameraRig set of cameras (fixed poses wrt body and intrinsics) taking the measurements
+   * @param params internal parameters of the smart factors
+   */
+  SmartProjectionPoseFactorRollingShutter(
+      const SharedNoiseModel& sharedNoiseModel, const Cameras& cameraRig,
+      const SmartProjectionParams& params = SmartProjectionParams())
+      : Base(sharedNoiseModel, params),
+        cameraRig_(cameraRig) {
+    // use only configuration that works with this factor
+    Base::params_.degeneracyMode = gtsam::ZERO_ON_DEGENERACY;
+    Base::params_.linearizationMode = gtsam::HESSIAN;
+  }
+
+  /**
+   * Constructor
+   * @param Isotropic measurement noise
+   * @param camera single camera (fixed poses wrt body and intrinsics)
    * @param params internal parameters of the smart factors
    */
   SmartProjectionPoseFactorRollingShutter(
-      const SharedNoiseModel& sharedNoiseModel,
+      const SharedNoiseModel& sharedNoiseModel, const Camera& camera,
```

Comment:
same comment, user can just call above constructor with {camera}

Reply:
this does not work. Maybe because a cameraSet is more than just a set of cameras?